### ISSUE TITLE
Pass config.active_support.test_order to minitest

### ISSUE THREAD

**Issue by phillipoertel** [blowmage/minitest-rails#205]
Wouldn't it be appropriate for this gem to pass the above setting to minitest? Otherwise `config.active_support.test_order` could be `:sorted` but the tests would still run in random order.
If I don't know a lot about Rails/Minitest, I'd expect to be able to set the test order with that one setting.

**Comment by zenspider** [comment]
I would expect that only to affect subclasses of active_support's test_case

**Comment by blowmage** [comment]
I think @zenspider is correct, this should already be handled by `ActiveSupport::TestCase`. I would expect tests that use `Minitest::Test` or `Minitest::Spec` to use their own settings/defaults.

**Comment by phillipoertel** [comment]
Ok, thanks for the clarification. I somehow thought that `minitest-rails` integrates minitest into Rails, forgetting that minitest is the default testing framework already. So `ActiveSupport::TestCase` should handle that if it wanted to, indeed.

**Comment by zenspider** [comment]
@phillipoertel well... `minitest-rails` DOES integrate minitest(/spec) into rails...

If you're using minitest/spec + rails + minitest-rails and you're NOT seeing random behavior, something wierd is going on and you should reopen w/ a reproduction of the bug if you can.

**Comment by phillipoertel** [comment]
It's actually the other way around :-). The tests are executed in random order, even when I set `config.active_support.test_order`to `:sorted`. I investigated the problem a little and decided it has to do with the heavily non-default test setup of the project, rather than minitest-rails. 

Since :random is the order we want anyhow, I'll document that `config.active_support.test_order` currently has no effect and point to this issue, if anyone ever wanted to change that.
Let me know if you suspect an issue in minitest-rails; then I'll help you debug. But I think the problem is on our side, and not worth the effort debugging right now.
